pkg/terraform/workspace.go

```diff
@@ -132,6 +134,7 @@ func (w *Workspace) Apply(ctx context.Context) (ApplyResult, error) {
 	}
 	cmd := exec.CommandContext(ctx, "terraform", "apply", "-auto-approve", "-input=false", "-lock=false", "-json")
 	cmd.Dir = w.dir
+	w.setChildEnv(cmd)
```

Comment:
nitpick

Quoting from comments of `Env` field:
> 	// Env specifies the environment of the process.
	// Each entry is of the form "key=value".
	// If Env is nil, the new process uses the current process's
	// environment.

Would it be more clear if we do the operation here?
```suggestion
	cmd = append(os.Environ(), w.credentialsEnv...)
```

Reply:
Moved cmd.Dir sets also to that function.